### PR TITLE
Add step_duration option to stdout exporter (issue #27)

### PR DESCRIPTION
**Pull request by uggla**
@bpetit, please let me know if you want a different description or switches.
Error handling is maybe a bit brutal by making a panic if the value passed is not a numeric one.

### CODE REVIEW

File: docs/exporters/stdout.md [hunk 1]
```diff
@@ -24,4 +28,5 @@ As always exporter's options can be displayed with `-h`:
         -V, --version    Prints version information
 
     OPTIONS:
-        -t, --timeout <timeout>    Maximum time spent measuring, in seconds. [default: 10]
\ No newline at end of file
+	    -s, --step <step_duration>    Set measurement step duration in second. [default: 2]
```
Comment (bpetit) [inline docs/exporters/stdout.md:31]: Could you just remove the extra tabulation please ?

**Comment by bpetit** [comment]
**Thanks a lot** for this PR, this is great ! 
I just noticed an extra tabulation in `docs/exporters/stdout.md`. Could you fix that prior to merge ?

File: docs/exporters/stdout.md (hunk 1 above)
Comment (uggla) [reply docs/exporters/stdout.md]: Should be ok now.

**Comment by bpetit** [comment]
This is great ! Thanks :)

**Pull request merged**